source/containers.tex

```diff
@@ -6338,7 +6338,7 @@
 \pnum
 \requires
 \tcode{is_assignable_v<mapped_type\&, M\&\&>} shall be \tcode{true}.
-\tcode{value_type} shall be \tcode{EmplaceConstructible} into \tcode{map}
+\tcode{value_type} shall be \tcode{Emplace\-Constructible} into \tcode{map}
```

Comment:
Can you try adding `EmplaceConstructible` to the global hyphenation hints (in `std.tex`) and see if that works? If the issue is just that it's an unknown word, then that's a more sustainable approach.